Badge/badge.css

```diff
@@ -0,0 +1,174 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root{
+    --gray:#3e4152;
+    --text-color:#007FFF;
+    --blue-bg:#023e8a;
+    --white-bg:#FAFAFA;
+    --dnd:#ae2012;
+    --offline:#7d8597;
+    --online:#008000;
+    --black-bg:#000814;
+}
+
+body{
+    font-family: 'Poppins', sans-serif;
+    margin: 2rem;
+    color:var(--gray);
+}
+h1{
+    font-size:3rem;
+}
+
+h1, h2, p{
+    margin: 0.625rem 0;
+    line-height: 1.2;
+}
+
+h2{
+    font-size: 2rem;
+}
+p{
+    font-size: 1.2rem;
+    padding: 0.8rem;
+}
+.badge-box{
+    border:1px solid var(--gray);
+    margin: auto;
+    width:80%;
+    padding: 1.5rem;
+    border-radius: 6px;
+    margin-bottom: 1.5rem;
+    display: flex;
+    flex-wrap: wrap;
+}
+.fas, .fab{
+    font-size: 2rem;
+    padding: 0.4rem;
+    color: var(--black-bg);
+}
```

Comment:
But this is the className of font-awesome. I don't think I can change the class name of these classes. Please let me know if you have any other alternatives

Reply:
Suppose you are using five different icons, so for that, there will be five classes provided by font-awesome, so instead of styling each of them, have a common class eg icon and use it throughout your component.

Reply:
Updated it. Thank you